omnibus/config/software/system-probe.rb

```diff
@@ -37,6 +39,7 @@
     copy "#{ENV['SYSTEM_PROBE_BIN']}/tcp-queue-length.c", "#{install_dir}/embedded/share/system-probe/ebpf/runtime/"
     copy "#{ENV['SYSTEM_PROBE_BIN']}/clang-bpf", "#{install_dir}/embedded/bin/clang-bpf"
     copy "#{ENV['SYSTEM_PROBE_BIN']}/llc-bpf", "#{install_dir}/embedded/bin/llc-bpf"
+    command "tar -xf #{ENV['SYSTEM_PROBE_BIN']}/minimized-btfs.tar.gz -C #{install_dir}/embedded/share/system-probe/ebpf/co-re/btf/"
```

Comment:
Akin to my comment https://github.com/DataDog/datadog-agent/pull/13962#discussion_r1007213261 I think it would be best to leave this as a tarball.